common/src/test/java/com/datastrato/gravitino/dto/util/TestDTOConverters.java

```diff
@@ -181,4 +185,42 @@ void testListPartitionConvert() {
     Assertions.assertEquals(values.length, listPartitionDTO.lists().length);
     Assertions.assertEquals(properties, listPartitionDTO.properties());
   }
+
+  @Test
+  void testPartitionDTOConvert() {
+    Literal<?> lower = Literals.stringLiteral("2008-08-08");
+    Literal<?> upper = Literals.stringLiteral("us");
+    Map<String, String> properties = Collections.singletonMap("key", "value");
+    RangePartition rangePartition =
+        (RangePartition) Partitions.range("range", upper, lower, properties);
```

Comment:
Can we modify the method `Partitions.range` method to directly return `RangePartition`?

Reply:
Looks good to me, I'll give it a try.

Reply:
This is already done.